changelog/3772.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed a bug with the calculation of Carrington longitude as seen from Earth where the true Sun-Earth distance was not properly considered.
+This bug resulted in errors no greater than 7 arcseconds in Carrington longitude when using `~sunpy.coordinates.sun.L0` and `~sunpy.coordinates.frames.HeliographicCarrington`.
```

Comment:
It makes me happy that this is the level of accuracy we are working to :grinning: